src/test/java/be/syntra/devshop/DevshopBack/services/utilities/AddressMapperUtilityTest.java

```diff
@@ -0,0 +1,54 @@
+package be.syntra.devshop.DevshopBack.services.utilities;
+
+import be.syntra.devshop.DevshopBack.entities.Address;
+import be.syntra.devshop.DevshopBack.models.AddressDto;
+import be.syntra.devshop.DevshopBack.testutilities.AddressUtils;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AddressMapperUtilityTest {
+    private AddressMapperUtility addressMapperUtility = new AddressMapperUtility();
+
+    @Test
+    void convertToAddress() {
+        // given
+        Address address = AddressUtils.createAddress();
+        AddressDto addressDto = AddressUtils.createAddressDto();
+
+        // when
+        Address mappedAddress = AddressMapperUtility.convertToAddress(addressDto);
+
+        // then
+        assertEquals(mappedAddress.getClass(), Address.class);
+        assertEquals(mappedAddress.getStreet(), address.getStreet());
+        assertEquals(mappedAddress.getNumber(), address.getNumber());
+        assertEquals(mappedAddress.getBoxNumber(), address.getBoxNumber());
+        assertEquals(mappedAddress.getPostalCode(), address.getPostalCode());
+        assertEquals(mappedAddress.getCity(), address.getCity());
+        assertEquals(mappedAddress.getProvince(), address.getProvince());
+        assertEquals(mappedAddress.getCountry(), address.getCountry());
```

Comment:
here you want to check if the result of the mapping (lappedAddress) has the same data as the thing you put in the mapper (addresDto) the first line is checking if the class is correct, thats good, but the rest should be a check between mappedAddress-fields and addresDto-fields, and the `Address address...` line should be removed from // given